pkg/registry/scripts/projects.js

```diff
@@ -85,33 +90,88 @@
         '$modal',
         'projectUtil',
         'projectActions',
-        function($scope, loader, select, $modal, util, projectAction) {
-            loader.watch("users");
-            loader.watch("groups");
-            loader.load("Project", null, null);
+        '$routeParams',
+        'ListingState',
+        '$location',
+        function($scope, loader, select, $modal, util, projectAction, $routeParams, ListingState, $location) {
+            var namespace = $routeParams["namespace"] || "";
+            if (namespace) {
+                $scope.listing = new ListingState($scope);
+                $scope.listing.single = true;
 
-            $scope.users = function() {
-                return select().kind("User");
-            };
+                loader.load("Project", null, null);
+                $scope.project = function() {
+                    return select().kind("Project").name(namespace).one();
+                };
 
-            $scope.groups = function() {
-                return select().kind("Group");
-            };
+            } else {
 
-            $scope.projects = function() {
-                return select().kind("Project");
-            };
+                $scope.listing = new ListingState($scope);
+                $scope.listing.single = false;
```

Comment:
Why is this set to false?

In addition the menu (upper right) has an 'Expand items inline' option. It has no effect. It should have an effect, and currently the default would be to not 'expand items inline' (ie: menu option not checked)